terraform/modules/pre-auth-lambda/lambda/lambda_function.py

```diff
@@ -0,0 +1,76 @@
+import os
+import datetime
+import dateutil
+from dateutil.relativedelta import relativedelta
+import boto3
+from boto3.dynamodb.conditions import Key
+
+tableName = os.environ.get("TABLE_NAME")
+regionName = os.environ.get("AWS_REGION")
```

Comment:
should handle the exception if these aren't set

Reply:
This is done, can you please review again after my commit to make sure the solution is good?